src/couch_httpd.erl

```diff
@@ -161,8 +161,15 @@ set_auth_handlers() ->
     AuthenticationSrcs = make_fun_spec_strs(
         config:get("httpd", "authentication_handlers", "")),
     AuthHandlers = lists:map(
-        fun(A) -> {make_arity_1_fun(A), ?l2b(A)} end, AuthenticationSrcs),
-    ok = application:set_env(couch, auth_handlers, AuthHandlers).
+        fun(A) -> {auth_handler_name(A), make_arity_1_fun(A)} end, AuthenticationSrcs),
+    AuthenticationFuns = AuthHandlers ++ [
+        {<<"local">>, fun couch_httpd_auth:local_admin_handler/1} %% should be last
+    ],
+    ok = application:set_env(couch, auth_handlers, AuthenticationFuns).
```

Comment:
hrm, this is unpleasant, application env is not persistent, are we using here as a cache?